tests/helper/helper_generic.go

```diff
@@ -311,3 +312,21 @@ func CommonAfterEach(commonVar CommonVar) {
 
 	os.Unsetenv("GLOBALODOCONFIG")
 }
+
+// GjsonMatcher validates if []results from gjson.GetMany match the expected values for each json path requested
+// Values is an array of results returned by the gjson.GetMany function
+// Expected is an array of strings, defines the expected values for each of the paths requested in the gjson.GetMany function
+// For documentation about gjson see https://github.com/tidwall/gjson#get-multiple-values-at-once
+func GjsonMatcher(values []gjson.Result, expected []string) bool {
+	matched := 0
+	for i, v := range values {
+		if strings.Contains(v.String(), expected[i]) {
+			matched++
+		}
+	}
+	numVars := len(expected)
+	if matched == numVars {
+		return true
+	}
+	return true
```

Comment:
We're returning `true` if `matched` and `numVars` are equal. We're returning `true` otherwise as well. Essentially this function will always return `true`. Is this by design or did we intend to have `return false` if the two vars aren't equal? If it's a mistake, we can replace:
```go
if matched == numVars {
    return true
}
return true
```
with
```go
return matched == numVars
```

Reply:
You are right, i had false on an earlier version. Replacing it as per your suggestion